api/ext_news/urls.py

```diff
@@ -2,7 +2,7 @@
 from .views import PostUpd, Post, MailingAPIView
 
 urlpatterns = [
-    path('upd/<int:pk>/', PostUpd.as_view(), name='upd'),
-    path('post/', Post.as_view(), name=None),
+    path('update/<int:pk>', PostUpd.as_view(), name='upd'),
```

Comment:
'update' not needed here, as for me. To have the same urls with other applicatios you have to use
/api/v1/ext_news/ - for massive update/delete
/api/v1/ext_news/<int:pk>/ - for edit by id

Reply:
update urls.py